openadapt/finetune.py

```diff
@@ -0,0 +1,52 @@
+from openadapt import crud, models
+from loguru import logger
+import fire
+
+VALID_MODELS = ("davinci", "mpt-7b")
+
+
+def main(recording_id: int, model_name: str):
+    finetune_recording = crud.get_recording_by_id(recording_id)
+    if not finetune_recording:
+        raise ValueError("A recording with the given id does not exist")
```

Comment:
```suggestion
    assert finetune_recording, "A recording with the given id does not exist"
```
not sure if this is better but its neat

Reply:
<img width="789" alt="image" src="https://github.com/OpenAdaptAI/OpenAdapt/assets/57018940/303429bb-a77a-40ef-a806-34a481e35bd9">

the precedent